test/backend_graph_comparison.in.cpp

```diff
@@ -185,7 +266,42 @@ NGRAPH_TEST_P(${BACKEND_NAME}, serialized_graph_files, compare_backends_with_gra
         }
     }
 
-    //no need to include original results they are subsumed by new_results
+    // Collect set of parameters and results to run test backend with isolated
+    // inputs which will be copied from the reference backend outputs
+    ParameterVector isolated_parameters = func->get_parameters();
+    NodeVector isolated_results;
+    unordered_map<ngraph::Node*, std::shared_ptr<ngraph::Node>> isolated_node_to_original_node;
+    for (auto n : func->get_ordered_ops())
+    {
+        // Don't include op::Results otherwise Function c-tor will complain
+        if (!n->is_output() && !n->is_parameter() && !n->is_constant() &&
+            !(n->get_outputs().size() > 1))
```

Comment:
I could see removing `!(n->get_outputs().size() > 1)`, and then checking if it is a multi output op. If it is, add a GOE for each output and then add that to the result list. That way if a graph is only using a few of the outputs, we still check all outputs of an op.

Reply:
After posting my last comment about multi-output ops, I realized that I think the isolated op testing may not cover multi-output ops at all currently. I think with this conditional you will only be testing the GOEs (essentially piping a CPU op output through a GPU GOE). If so, we need to handle multi-output ops with an explicit conditional inside this scope before merge. Can you check? A graph that has op::BatchNormTraining in it would work.